routes/cluck.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express');
+const knex = require('../db/client');
+const router = express.Router()
+
+const trending = (content) => {
```

Comment:
ideally, these functions should live outside routes folder. you can create something like "helpers" or "utils" folder for functions like these.